package.json

```diff
@@ -139,13 +141,19 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.12.1",
     "@babel/template": "^7.10.4",
+    "@cfaester/enzyme-adapter-react-18": "^0.8.0",
```

Comment:
i think this is a temp solution right? Should we create an issue to migrate away from enzyme as well as removing this?